cmd/.krew.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: commatrix
+spec:
+  version: v0.0.2
```

Comment:
This is the first version we release why not starting from v0.0.1?

Reply:
We already have 0.0.1

Reply:
Can we update it to match the krew index?